build/DependencyVersions.props

```diff
@@ -10,7 +10,7 @@
     <MsBuildPackagesVersion>15.2.0-preview-000093-02</MsBuildPackagesVersion>
     <DependencyModelVersion>2.0.0-preview1-001960</DependencyModelVersion>
     <PlatformAbstractionsVersion>2.0.0-preview1-001960</PlatformAbstractionsVersion>
-    <NuGetVersion>4.3.0-preview1-2500</NuGetVersion>
+    <NuGetVersion>4.3.0-preview1-4045</NuGetVersion>
```

Comment:
Yeah, I called that out.  It's needed for the test coverage, so we could split it out if needed.